src/components/kyb/FinancialInformationStep.tsx

```
@@ -0,0 +1,239 @@
+  };
+
+  const validateField = (field: string) => {
+    const value = data[field as keyof BusinessData] as string;
+    const newErrors = { ...errors };
+
+    switch (field) {
+      case 'annualRevenue':
+        if (!value) {
+          newErrors.annualRevenue = 'Annual revenue is required';
+        } else {
+          delete newErrors.annualRevenue;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)